File: Code/田正东/田正东第1周/k.c
#include <stdio.h>
#include <math.h>
int main()
{
    int x,y,a,b,i,j,flag;
	while(scanf("%d %d",&x,&y)!=EOF)
	{    
		a=0;
	    if(x==0&&y==0)
		{
		    break;
		}
		for(i=x;i<=y;i++)
		{
		     flag=1;
			 b=i*i+i+41;
			 for(j=2;j<=sqrt(b);j++)
			 {
			     if(b%j==0)
				 {
				     flag=0;
					 break;
				 }
			 }
			 if(flag)
			 {
			     a++;
			 }
		}
		if(a==y-x+1)
		{
		    printf("OK\n");
		}
		else
		{
		    printf("Sorry\n");
		}
	}
	return 0;
}

File: Code/贺含悦/VJudge/week1/K.java
/*************************************************************************
	> File Name: K.java
	> Author: 
	> Mail: 
	> Created Time: 2018年07月24日 星期二 02时27分38秒
 ************************************************************************/
import java.util.*;
public class K {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while(sc.hasNext()) {
            int a = sc.nextInt();
            int b = sc.nextInt();
            if(a == 0 && b == 0) {
                break;
            }
            boolean flag = true;
            for(int i = a; i <= b; i++) {
                if(!isPrime(i*i+i+41)) {
                    flag = false;
                    break;
                }
            }
            if(flag) {
                System.out.println("OK");
            } else {
                System.out.println("Sorry");
            } 
        }
    }
    public static boolean isPrime(int n) {
        for(int i = 2; i <= Math.sqrt(n); i++) {
            if(n % i == 0) {
                return false;
            }
        }
        return true;
    }
}

